src/encoded/static/components/matrix_entex.js

```diff
@@ -106,24 +129,36 @@ const DonorLegend = () => (
  * in the matrix. Any determination of column order or inclusion/exclusion happens in this
  * function. Do not rely on the order of the keys. The resulting object has the form:
  * {
- *   'cat1': {col: 0, category: 'cat1', hasSubcategories: true},
+ *   'cat1': {col: 0, category: 'cat1'},
  *   'cat1|subcat1': {col: 1, category: 'cat1', subcategory: 'subcat1'},
  *   'cat1|subcat2': {col: 2, category: 'cat1', subcategory: 'subcat2'},
```

Comment:
I have no seems anyone use pipe in a variable name

Reply:
It’s a key, not a variable name.